scss/_base/_forms.scss

```diff
@@ -1,59 +1,99 @@
+// A label and a form element.
+//
+// .-padded - Adds extra padding above and below field group.
+// .-inline - Form elements appear side-by-side.
+//
+// Styleguide Forms - Field Groups
+.field-group {
+  margin-bottom: ($base-spacing / 2);
```

Comment:
My concern is that wrapping markup (of the sort Drupal likes to contribute) might make this really frustrating in the future. For example, half of the fields in the registration form are wrapped in an invisible fieldset that we can't get rid of.